data/remnant/remnant missions.txt

```diff
@@ -875,10 +879,63 @@ mission "Remnant: Technology Available"
+			`	The old man smiles. "While we might not have access to modern licenses, we do have all our old entertainment programs from before the exile. Some of them included details such as what licenses looked like. While things might have changed in the intervening years, licenses were historically governed by petty bureaucracy, and those are very resistant to change. So it seemed very likely that things have not changed much. And definitely not so much as to resemble ours." He seems amused, but then his smile fades.`
+			`	"Then again, ancestral space never designed their licenses to withstand much. They are incentivized to not be so secure, are they not? Enough money and the right connections, and licenses for anything can be acquired. These" he holds up a Remnant license "not so much." He shakes his head and becomes serious. "Now, your license has a handful of connection pads on the bottom, which can be connected to your ship's IFF. When you are inbound for landing, simply tap this code on the license, and the cube will verify the pattern as well as your fingerprints, a few other biometrics to ensure you are indeed living and concious, and then transmit an encrypted code. Our IFF systems will verify the transmitted signal and respond accordingly."`
+			choice
+				`	"That sounds thorough."`
+					goto thorough
+				`	"So why haven't I been shot down yet?"`
+					goto shotdown
+				`	"What is an 'IFF' system?"`
+			`	"You aren't familiar with IFF?" He asks, looking slighly incredulous. "IFF stands for Identify Friend or Foe, and is the general name for the system that identifies your ship to those around it. It is what determines which ships should be highlighted red or orange on your system map, and provides limited movement detail to allied ships so that targetting systems can fire past you. I do not know about their current practices, but historically the Navy and most other organizations had complex methods of verifying the identify of other ships. So it is not simply a case of having your IFF state that you are a friendly."`
+			choice
+				`	"Thanks for the explanation."`
+					goto continue
+				`	"That sounds thorough."`
+					goto thorough
+				`	"So why haven't I been shot down yet?"`
+			label shotdown
+			`	"Because every system has potential failures and vulnerabilities, we ensure that people are always involved in the decision making process. Our initial scans showed that you were not likely an Alpha, and we were curious as to who you were and what opportunities might come of it. Since then the various monitors of the system have had to manually designate you as 'friendly' every time you made an approach. It worked, for a time. But now it is time to make your acceptance official. While our monitors are effective, it still wastes time to force them to track you and cancel the alerts every time you approach. Since you are trustworthy, better to give you the means to identify yourself so our defences can focus on real threats."`
```

Comment:
```suggestion
			`	"Because every system has potential failures and vulnerabilities, we ensure that people are always involved in the decision making process. Our initial scans showed that you were not likely an Alpha, and we were curious as to who you were and what opportunities might come of it. Since then the various monitors of the system have had to manually designate you as 'friendly' every time you made an approach. It worked, for a time. But now it is time to make your acceptance official. While our monitors are effective, it still wastes time to force them to track you and cancel the alerts every time you approach. Since you are trustworthy, better to give you the means to identify yourself so our defenses can focus on real threats."`
```